docs/usage/overview.md

```diff
@@ -3,10 +3,11 @@
 This section outlines methods of using of the database, and goes through some examples.
 
 There are several methods of using the database, depending on your purpose:
-- [**General Browsing/Querying**](#general-browsingquerying): Use an application such as [DBeaver](https://dbeaver.io/) to query the  [SQLite database](https://github.com/ShabadOS/database/releases)
-- [**Offline App**](#offline-applications-sqlite): Download the [SQLite database](https://github.com/ShabadOS/database/releases)
-- [**Offline JavaScript App (recommended)**](#offline-applications-javascript): Install the [API via npm](installation#npm)
-- [**Online App**](#online-applications-gurbaninow-api): Use the [GurbaniNow API](https://github.com/GurbaniNow/api)
+- [Overview](#overview)
+  - [General Browsing/Querying](#general-browsingquerying)
```

Comment:
Btw I'd always put commits like this that are unrelated in a separate PR (branched cleanly off `dev`) so that if there's a disagreement on the unrelated changes, the PR doesn't get stuck.